TestSequencePhenotype.java

```diff
@@ -111,4 +111,18 @@ public void testToString() {
 		assertEquals("ACGT", simplePheno.toString());
 	}
 
+	/**
+	 * Test equals()
+	 */
+	@Test
+	public void testEquals() {
+		// Same SequencePhenotype objects are equal
+		assertTrue(emptyPheno.equals(emptyPheno));
+
+		SequencePhenotype simplePhenoTest = new SequencePhenotype("ACGT");
+
+		// SequencePhenotype objects with the same sequence are equal
+		assertTrue(simplePheno.equals(simplePhenoTest));
```

Comment:
Maybe a test to ensure `SequencePhenotype.equals()` returns false whenever the sequences are different?